### PR TITLE
24w37a ender pearls

### PR DESCRIPTION
**Pull request by EliTheCoder**
Some new methods that have to do with ender pearl chunk loading.

This is my first contribution and I have read CONTRIBUTING.md and CONVENTIONS.md but please let me know if anything looks off.

### CODE REVIEW

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/entity/projectile/thrown/EnderPearlEntity.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/unmapped/C_xmtgsyuv net/minecraft/entity/projectile/thrown/EnderPearlEntity
+	FIELD f_hbarhomd chunkLoadTimer J
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/projectile/thrown/EnderPearlEntity.mapping]:
```suggestion
	FIELD f_hbarhomd chunkLoadTicks J
```

File: mappings/net/minecraft/server/network/ServerPlayerEntity.mapping [hunk 2]
```diff
@@ -183,6 +184,18 @@ CLASS net/minecraft/unmapped/C_mxrobsgg net/minecraft/server/network/ServerPlaye
 	METHOD m_yhuimfko gameModeFromNbt (Lnet/minecraft/unmapped/C_hhlwcnih;Ljava/lang/String;)Lnet/minecraft/unmapped/C_lghcpyvl;
 		ARG 0 nbt
 		ARG 1 key
+	METHOD m_cqxrhbhc addEnderPearlTicket (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_ynrszrtu;)J
+		COMMENT Adds a chunk ender pearl chunk ticket to the ticket manager.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/server/network/ServerPlayerEntity.mapping]:
```suggestion
		COMMENT Adds an ender pearl chunk ticket to the ticket manager.
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]
looks good, just nitpicks

**Comment by EliTheCoder** [comment]
Perfect, thanks

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**